DEVELOPMENT.md

```diff
@@ -0,0 +1,20 @@
+Developer docs
+==================================
+
+## 📝 Prerequisites
+* ffmpeg
```

Comment:
ffmpeg (minimum version 4, version 5 recommended), must be available in the path so install at OS level.